### ISSUE TITLE
Nice CSS Extraction 

### ISSUE THREAD

**Issue by ncammarata** [flintjs/flint#120]
```js
$count = { .. }
```
inside view Button should extract the static styles and create the selector

```
  .ButtonView count, .ButtonView .count {  ... }
```

**Comment by ncammarata** [comment]
This also solves our precedence issues by offloading it to the browser

**Comment by natew** [comment]
Released in flint-js@1.3.116

We have some issues though that we need to go over, namely dynamic styles will always override static. Not sure best way to resolve that.
